k8s/test/e2e/local-assembly-container/Makefile

```diff
@@ -0,0 +1,27 @@
+# Copyright 2020-2022 Alibaba Group Holding Limited.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+REGISTRY := docker.pkg.github.com/v6d-io/v6d
+TAG      := latest
+
+all: build-local-assembly docker-push
```

Comment:
Why do you need to push the image? It looks only used in the CI....

Reply:
If it won't be used by end-user and only used by CI, please don't push.

Reply:
OK, I know your meaning, the image should be fixed and can only be pushed once as.